### ISSUE TITLE
Build Error on release 2.0.3

### ISSUE THREAD

**Issue by amirhmoradi** [chemerisuk/cordova-plugin-firebase-config#14]
Hi and thanks for your great cordova plugins !

After updating to 2.0.3, I get a build error (details below).

### For info, here is my env

Node version: v12.3.1
Cordova version: 8.1.1
Cordova Android Platform: 7.1.4
Gradle version: 5.4.1

Available Android targets: 
  * id: 1 or "android-27"
     Name: Android API 27
     Type: Platform
     API level: 27
     Revision: 3

  * id: 2 or "android-28"
     Name: Android API 28
     Type: Platform
     API level: 28
     Revision: 6

### Console output
```
...
:app:generateDebugSources
:app:javaPreCompileDebug
:app:compileDebugJavaWithJavac
/Users/user/workspace/project/mobileapp/platforms/android/app/src/main/java/by/chemerisuk/cordova/firebase/FirebaseConfigPlugin.java:60: error: cannot find symbol
        this.firebaseRemoteConfig.activate()
                                 ^
  symbol:   method activate()
  location: variable firebaseRemoteConfig of type FirebaseRemoteConfig
/Users/user/workspace/project/mobileapp/platforms/android/app/src/main/java/by/chemerisuk/cordova/firebase/FirebaseConfigPlugin.java:76: error: cannot find symbol
        this.firebaseRemoteConfig.fetchAndActivate()
                                 ^
  symbol:   method fetchAndActivate()
  location: variable firebaseRemoteConfig of type FirebaseRemoteConfig
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
 FAILED
```

**Comment by chemerisuk** [comment]
@amirhmoradi I guess you specified `FIREBASE_CONFIG_VERSION` other than default value `17.0.+`

**Comment by amirhmoradi** [comment]
thanks for your fast reply :)

I used the default version, it failed.
I tried version 16.4.1 as recommended here: https://github.com/arnesson/cordova-plugin-firebase/issues/1057#issuecomment-490000068
but failed again.
I reverted back your plugin to version 1.2.1 and build is successful.

By the way, I am using multiple plugins of yours, is there a way to manage all the variables you use ( `FIREBASE_*_VERSION` ) in a centralized way between your plugins?
